datanode/gateway/graphql/schema.graphql

```diff
@@ -380,6 +382,38 @@ type ObservableLiquidityProviderFeeShare {
   averageScore: String!
 }
 
+"The SLA statistics for each liquidity provider"
+type LiquidityProviderSLA {
+  "The liquidity provider party ID"
+  party: Party!
+  "Indicates how often LP meets the commitment during the current epoch."
+  currentEpochFractionOfTimeOnBook: String!
+  "Indicates how often LP meets the commitment during last epoch."
+  lastEpochFractionOfTimeOnBook: String!
+  "Indicates the fee penalty amount applied in the previous epoch."
+  lastEpochFeePenalty: String!
+  "Indicates the bond penalty amount applied in the previous epoch."
+  lastEpochBondPenalty: String!
+  "Holds the fee penalties from past epochs."
+  hysteresisPeriodFeePenalties: [String!]
```

Comment:
Is this for all LPs as a collective or per LP?

Reply:
Everything is per LP